bucket/wsltty.json

```
@@ -1,15 +1,15 @@
 {
+    "description": "Mintty as a terminal for Bash on Ubuntu on Windows / WSL",
     "version": "3.0.1.2",
     "license": "GPL-3.0-or-later",
     "url": "https://github.com/mintty/wsltty/releases/download/3.0.1.2/wsltty-3.0.1.2-install.exe#/dl.7z",
     "homepage": "https://github.com/mintty/wsltty",
     "hash": "8ba179890a47e41794f47dad674f92dbc747fefef3e6321bcef3956687914088",
-    "checkver": {
-        "github": "https://github.com/mintty/wsltty"
-    },
+    "checkver": "github",
     "autoupdate": {
         "url": "https://github.com/mintty/wsltty/releases/download/$version/wsltty-$version-install.exe#/dl.7z"
     },
+    "pre_install": "(\"@cd %~dp0\", (Get-Content \"$dir\\install.bat\")) | Out-File -Encoding ASCII \"$dir\\install.bat\"",
     "installer": {
         "file": "install.bat",
```

There are args missing.

https://github.com/lukesampson/scoop-extras/pull/1325#issuecomment-430950482

```suggestion
        "file": "install.bat",
        "args": [
            "\"$dir\"",
            "\"$dir\\config\"",
        ],
```

---

Then persisting of config is needed